### PR TITLE
PMM-5778 Check exporter format

### PR DESCRIPTION
**Pull request by askomorokhov**
https://jira.percona.com/browse/PMM-5778

FB: https://github.com/Percona-Lab/pmm-submodules/pull/1521

Ref:
 - https://github.com/percona/pmm/pull/688
 - https://github.com/percona/pmm-agent/pull/237
 - https://github.com/percona/pmm-managed/pull/670

### CODE REVIEW

**Review by AlekSi — CHANGES_REQUESTED** [review]
Unclear field name, no documentation comment

File: api/agentpb/agent.proto [hunk 1]
```diff
@@ -310,6 +310,8 @@ message CheckConnectionResponse {
   // Error message if connection check failed.
   string error = 1;
   Stats stats = 2;
+  // Contains the body of HTTP response of connection check to an exporter.
+  string exporter_response_body = 3;
```
Comment (AlekSi) [inline api/agentpb/agent.proto:314]: `CheckConnectionResponse` is about checking connectivity with Services, not with exporters. The word "exporter" is not used in this file at all.

Additionally, protobuf `string` can contain only UTF-8 data. I don't think we want limit us it it there

**Review by AlekSi — CHANGES_REQUESTED** [review]

File: api/agentpb/agent.proto (hunk 1 above)
Comment (askomorokhov) [reply api/agentpb/agent.proto:314]: @AlekSi, and what is your suggestion?

Comment (AlekSi) [reply api/agentpb/agent.proto:314]: https://github.com/percona/pmm/pull/689

Comment (askomorokhov) [reply api/agentpb/agent.proto:314]: Thanks, elegant detour :)

File: api/agentpb/agent.proto [hunk 2]
```diff
@@ -310,6 +310,8 @@ message CheckConnectionResponse {
   // Error message if connection check failed.
   string error = 1;
   Stats stats = 2;
+  // Contains the body of HTTP response of connection check to an exporter.
```
Comment (BupycHuk) [inline api/agentpb/agent.proto:313]: to be honest it looks weird for me that all other connections we check on pmm-agent side (even returning table_count), but for exporters we move connection check to pmm-server side.

**Pull request closed without merging**